pdc.sh

```diff
@@ -0,0 +1,27 @@
+RUSTPYTHONPATH=Lib
+export RUSTPYTHONPATH
+
```

Comment:
What is this script? What does pdc stand for? This does seem useful, I'm just not sure exactly what it does.

Reply:
PDC is an acronym for Pre-Delivery Check, as the intension is to run all checks before delivering (or raising a PR or commiting or what ever you want to use it for). 

Basically it executes all of the configured actions in the given order and reports which of them failed, without stopping in case of an fail. The actions in the given version are the RustPython CI steps plus a debug build in advance. So can run all of the checks in advance with one command.

Reply:
Will you put this comment on the top of this file as comment?